types/mytime.go

```diff
@@ -25,7 +25,7 @@ type MysqlTime struct {
 	month uint8  // month <= 12
```

Comment:
I think we should:
1. add a comment for this struct and warn "do no change the layout of add new fields"
2. add tests to make sure the layout is what we wish